docs/spec/spec.md

```diff
@@ -129,9 +129,9 @@ operation. This will close the corresponding connection pool if it is not shared
     public isolated function close() returns Error?;
    ```
 
-# 3. Queries and Values
+# 3. Queries and values
 
-## 3.1. ParameterizedQuery and Values
+## 3.1. ParameterizedQuery and values
```

Comment:
```suggestion
## 3.1. ParameterizedQuery and values
```
This should be either `ParameterizedQuery` (within backticks) or Parameterized Query